src/reducers/todosReducer.ts

```diff
@@ -27,6 +33,18 @@ const todosReducer = (state = initialState, action: TodoActionTypes): ITodoState
         data: [],
         error: action.payload
       };
+    case TOGGLE_TODO: {
```

Comment:
just a suggestion for the future, things like toggling, showing modal, and so on can be implemented in the local component state with a corresponding hook (`useState()`). We don't need to keep such things in the redux store in most of the cases.